templates/flutter/lib/src/client_browser.dart.twig

```diff
@@ -128,6 +133,23 @@ class ClientBrowser extends ClientBase with ClientMixin {
 
   @override
   Future webAuth(Uri url) {
-    throw UnimplementedError();
-  }
+  return FlutterWebAuth.authenticate(
+      url: url.toString(),
+      callbackUrlScheme: "appwrite-callback-" + config['project']!,
+    ).then((value) async {
+      Uri url = Uri.parse(value);
+      final key = url.queryParameters['key'];
+      final secret = url.queryParameters['secret'];
+      if (key == null || secret == null) {
+        throw AppwriteException(
+            "Invalid OAuth2 Response. Key and Secret not available.", 500);
+      }
+      Cookie cookie = new Cookie(key, secret);
+      cookie.domain = Uri.parse(_endPoint).host;
+      cookie.httpOnly = true;
+      cookie.path = '/';
+      List<Cookie> cookies = [cookie];
+      await init();
+      _cookieJar.saveFromResponse(Uri.parse(_endPoint), cookies);
```

Comment:
As this is web, we don't need to handle cookie ourselves, we can just remove this, with this we will not need to import cookie jar

Reply:
Deleted cookie jar and dart io altogether